assets/stylesheets/rolodex/components/_buttons.sass

```diff
@@ -132,6 +144,22 @@
     position: relative
     float: left
 
+    &.btn-style
+      background-color: $btn-group-bg
+      border-color: $btn-group-border-color
+      border-style: $btn-group-border-style
+      +rem(border-width, $btn-group-border-width)
+      color: $btn-group-color
+
+      &:hover
+        background-color: $btn-group-hover-bg
+        color: $btn-group-hover-color
+
+      &.active
+        background-color: $btn-group-active-bg
+        background-image: linear-gradient(-180deg, rgba(255,255,255,0.00) 0%, rgba(0,0,0,0.20) 100%)
```

Comment:
Nit: Add a space after each comma, a single zero in place of `0.00`, and `.2` in place of `0.20`.